sys-devel/gcc/gcc-5.4.0_2016_06_04.recipe

```diff
@@ -336,6 +363,25 @@ REQUIRES_syslibs="
 	haiku$secondaryArchSuffix
 	"
 
+# ----- gfortran package -----------------------------------------------
+
+SUMMARY_gfortran="C/C++-runtime static libraries and C++ headers, needed for gfortran"
+DESCRIPTION_gfortran="This package is not of general interest - it \
+contains the set of gcc's C/C++-runtime libraries and headers that is \
+required by gfortran to build fortran source code."
+
+PROVIDES_gfortran="
+	gcc${secondaryArchSuffix}_gfortran_devel = $portVersion compat >= 5
```

Comment:
Shouldn't this be changed to:
```
	gcc${secondaryArchSuffix}_gfortran = $portVersion compat >= 5
```
(i.e. without the _devel suffix) since we' re not defining PROVIDES_gfortran_devel but PROVIDES_gfortran ?